regulations.tex

```diff
@@ -3,58 +3,9 @@ \section{Regulations}
 
 \begin{enumerate}
 \item The Judge's decision is final.
-\item All robots must be safe.
-  \begin{enumerate}
-    \item This is defined considering safety concerns including, but not limited
-          to:
-      \begin{enumerate}
-        \item sharp edges;
-        \item the effects of impact at speed;
-        \item fire risks from the battery (see Regulation~\ref{reg:lipo}).
-      \end{enumerate}
-    \item No robots will be permitted to compete without passing a safety and
-          compliance inspection.
-    \item \staff may reinspect your robot and invalidate previous inspections at
-          any time.
-  \end{enumerate}
 \item Any assistance from \staff is provided without guarantees.
 \item Competitors are expected to behave within the spirit of good
       sporting conduct.
-\item Competitors must take reasonable measures to avoid their robot damaging
-      the arena, or anything within it, including other robots. This is a
-      non-contact sport.
```

Comment:
We should probably still have some kind of regulation about being a non-contact sport

Reply:
c08924f4680cf26f4c37cf91732620dfc3ea918f